platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/platformstatus/PlatformStatusEvent.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.common.system.platformstatus;
+
+/**
+ * An enum representing events that can occur which may have an impact on the {@link PlatformStatus}
+ */
+public enum PlatformStatusEvent {
```

Comment:
I advise caution using "event" in the name of something, if it can be avoided, since everywhere else "event" has a very specific meaning. Perhaps "incident" or "action" or something similar?

Reply:
I originally had `occurrence`, but @poulok suggested to use `event`. I see the concern, I think I'll switch to `action` for now, unless consensus lands elsewhere